frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewMetaData/FeatureOverviewMetaData.tsx

```diff
@@ -25,24 +76,29 @@ const FeatureOverviewMetaData = () => {
     const IconComponent = getFeatureTypeIcons(type);
 
     return (
-        <div className={classnames(styles.container)}>
-            <div className={styles.paddingContainerTop}>
-                <div className={styles.metaDataHeader} data-loading>
-                    <IconComponent className={styles.headerIcon} />{' '}
-                    <h2 className={styles.header}>
-                        {capitalize(type || '')} toggle
-                    </h2>
-                </div>
-                <div className={styles.body}>
-                    <span className={styles.bodyItem} data-loading>
+        <StyledContainer>
+            <StyledPaddingContainerTop>
+                <StyledMetaDataHeader data-loading>
+                    <IconComponent
+                        sx={{
+                            marginRight: '1rem',
+                            height: '40px',
+                            width: '40px',
+                            fill: '#fff',
```

Comment:
Can we use the theme here? We can't implement any other theme modes until all colors are directly controlled by the theme.